web/packages/teleport/src/Notifications/notificationContentFactory.tsx

```diff
@@ -26,6 +26,7 @@ import {
 } from 'teleport/services/notifications';
 import { Label } from 'teleport/types';
 
+// notificationContentFactory processes the content for notifications for OSS features.
```

Comment:
`processes` is a bit unclear, I'd replace it with `produces` (in e too).

I'd also switch the comment to JSDoc.